sdk/storage/azure-storage-file-datalake/tests/test_file_system.py

```diff
@@ -203,9 +203,9 @@ def test_rename_file_system(self, datalake_storage_account_name, datalake_storag
             self.dsc._rename_file_system(name="badfilesystem", new_name="filesystem")
         self.assertEqual(new_name, new_filesystem.get_file_system_properties().name)
 
+    @pytest.mark.skip(reason="Feature not yet enabled. Record when enabled.")
```

Comment:
Hmm, at this point I find it hard to believe it will be enabled if it's not yet. Possibly this feature was cut and never actually shipped. We should check with Sean probably.

Reply:
Will leave this open for now and check back with Sean tomorrow when he is back 😄

Reply:
Update: This feature has been problematic and is pending to be enabled (possibly) in the future.. so continuing to skip for now 😄